frameworks/engineering/backend.md

```diff
@@ -206,6 +206,11 @@ topics:
           - criteria: "Implements complex RPC handlers"
             examples:
               - "Implements complex business logic, orchestrates multi-stage processes, deals with idempotency or distributed data sources"
+          - criteria: "Proactively considers security implications of their work"    
+            examples:
+              - "Appropriately ties down internal access to resources they're working with (e.g. RPC blacklisting, BigQuery permissions)"
+              - "\"There's a risk here of X, Y but given Z, I think this is OK\""
+              - "Loops in Security when their squad is unsure about something"
```

Comment:
Small thing: would be good to specify how someone loops in security. Imagine you're reading this as a new starter!

Reply:
Makes sense, I just pushed https://github.com/monzo/progression-framework/pull/54/commits/3c0435e069ae175f3234717812d2a05a934ab96b to address this.